packages/usa-character-count/src/usa-character-count.twig

```diff
@@ -8,7 +8,10 @@
         id="with-hint-input"
         maxlength="25"
         name="with-hint-input"
-        aria-describedby="with-hint-input-info with-hint-input-hint">
+        aria-describedby="with-hint-input-info with-hint-input-hint"
+        {% if disabled_state == 'disabled' %} disabled {%- endif %}
```

Comment:
Since `aria-disabled` is an option of `disabled_state`, this will enable `disabled` and `ariaDisabled` at the same time. The default `disabled_state` is `none`, so `disabled_state` will always come back as true because there's always a value.

I could switch the default value to `false`, but if we're just checking for `disabled_state` we can't switch between `disabled` and `aria_disabled`

I could do 
```twig
{% if disabled_state == 'disabled' %} disabled
{% elseif disabled_state == 'aria_disabled' %} aria-disabled="true" {%- endif %}
```
If we prefer it to be an if/else